buildbot/osuosl/master/config/builders.py

```diff
@@ -1848,6 +1824,34 @@
                         '-DLLVM_ENABLE_LLD=ON',
                     ])},
 
+# Build the LLVM dylib .so with all backends and link tools to it
+    {'name' : 'llvm-dylib-all',
+    'tags'  : ['llvm'],
+    'collapseRequests': False,
+    'workernames': ['gribozavr4'],
+    'builddir': 'llvm-dylib-all',
+    'factory' : UnifiedTreeBuilder.getCmakeWithNinjaBuildFactory(
+                    clean=True,
+                    depends_on_projects=['llvm'],
+                    checks=['check-all'],
+                    extra_configure_args=[
+                        '-DCMAKE_BUILD_TYPE=Release',
+                        '-DCMAKE_C_COMPILER=/usr/bin/gcc',
+                        '-DCMAKE_CXX_COMPILER=/usr/bin/g++',
+                        '-DLLVM_ENABLE_ASSERTIONS=On',
+                        '-DLLVM_BUILD_EXAMPLES=Off',
+                        "-DLLVM_LIT_ARGS=-v --xunit-xml-output test-results.xml",
+                        '-DLLVM_CCACHE_BUILD=On',
+                        '-DLLVM_CCACHE_DIR=/ccache',
+                        '-DLLVM_CCACHE_MAXSIZE=20G',
```

Comment:
Please remove the CCACHE flags and DCMAKE_C_COMPILER. Instead, please imitate the configuration of the builder that you're replacing (environment variables).